From 0b062d3b8bfd93bb58c387d5f9d21f70181ec301 Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.corp-partner.google.com>
Date: Thu, 18 Jun 2026 06:20:26 +0000
Subject: [PATCH] Add support for Wildcat Lake

This patch adds supports for wcl in intel's minigbm.

Tracked-On: NA
Signed-off-by: Tekriwal, Tanuj <tanuj.tekriwal@intel.corp-partner.google.com>
---
 xe.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/xe.c b/xe.c
index 7030b23..9044b6d 100644
--- a/xe.c
+++ b/xe.c
@@ -103,6 +103,8 @@ static void xe_info_from_device_id(struct xe_device *xe)
 
 	const uint16_t ptl_ids[] = { 0xB080, 0xB090, 0xB0A0, 0xB0B0, 0xB0FF};
 
+	const uint16_t wcl_ids[] = { 0xfd80, 0xfd81 };
+
 	unsigned i;
 	xe->graphics_version = 0;
 	xe->is_xelpd = false;
@@ -142,6 +144,12 @@ static void xe_info_from_device_id(struct xe_device *xe)
 			xe->graphics_version = 12;
 			xe->is_mtl_or_newer = true;
 		}
+
+	for (i = 0; i < ARRAY_SIZE(wcl_ids); i++)
+		if (wcl_ids[i] == xe->device_id) {
+			xe->graphics_version = 12;
+			xe->is_mtl_or_newer = true;
+		}
 }
 
 static void xe_get_modifier_order(struct xe_device *xe)
-- 
2.34.1

